.idea/copyright/rls.xml

```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
+  <copyright>
+    <option name="notice" value="Copyright (c) &amp;#36;today.year. REAL LIFE SCIENCES INC" />
```

Comment:
Does it really make sense to include a RLS copyright statement here? I am not an Intellij user, so I don't know what this is used for, but I have the feeling that this could (should?) be removed?